go/proto/ctrl/v1/secrets.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package ctrl.v1;
+
+option go_package = "github.com/unkeyed/unkey/go/gen/proto/ctrl/v1;ctrlv1";
+
+// SecretsConfig is stored in the deployments table
+// Contains encrypted environment variables snapshotted at deploy time
+message SecretsConfig {
+  // key -> encrypted value
+  map<string, string> secrets = 1;
```

Comment:
hmm, the value here is what exactly? none+ciphertext?  
  
why don't we use a `map<string, encrypted>`  
  
`￼message encrypted {￼``  
``  string nonce = 1;``  
``  string ciphertext = 2;``  
``}￼​​`​  
  
or bytes instead of strings

Reply:
ah ok I see this is just the `encrypted` value that gets returned by vault, let's keep treating that as opaque string